example/standalone-vault-provided-creds/main.tf

```diff
@@ -0,0 +1,124 @@
+module "minio" {
+  source = "github.com/fredrikhgrelland/terraform-nomad-minio.git?ref=0.3.0"
+
+  # nomad
+  nomad_datacenters = ["dc1"]
+  nomad_namespace   = "default"
+  nomad_host_volume = "persistence-minio"
+
+  # minio
+  service_name    = "minio"
+  host            = "127.0.0.1"
+  port            = 9000
+  container_image = "minio/minio:latest" # todo: avoid using tag latest in future releases
+
+  # user provided  credentials
+  vault_secret = {
+    use_vault_provider      = true,
+    vault_kv_policy_name    = "kv-secret",
+    vault_kv_path           = "secret/data/random-string/minio",
+    vault_kv_access_key     = "access_key",
+    vault_kv_secret_key     = "secret_key"
+  }
+
+  data_dir                        = "/minio/data"
+  buckets                         = ["default", "hive"]
+  container_environment_variables = ["JUST_EXAMPLE_VAR1=some-value", "ANOTHER_EXAMPLE2=some-other-value"]
+  use_host_volume                 = false
+  use_canary                      = true
+
+  # mc
+  mc_service_name                    = "mc"
+  mc_container_image                 = "minio/mc:latest" # todo: avoid using tag latest in future releases
+  mc_container_environment_variables = ["JUST_EXAMPLE_VAR3=some-value", "ANOTHER_EXAMPLE4=some-other-value"]
+}
+
+module "postgres" {
+  source = "github.com/fredrikhgrelland/terraform-nomad-postgres.git?ref=0.3.0"
+
+  # nomad
+  nomad_datacenters = ["dc1"]
+  nomad_namespace   = "default"
+  nomad_host_volume = "persistence-postgres"
+
+  # postgres
+  service_name                    = "postgres"
+  container_image                 = "postgres:12-alpine"
+  container_port                  = 5432
+  vault_secret                    = {
+    use_vault_provider     = true,
+    vault_kv_policy_name   = "kv-secret",
+    vault_kv_path          = "secret/data/random-string/postgres",
+    vault_kv_username_name = "username",
+    vault_kv_password_name = "password"
+  }
+  admin_user                      = "hive"
+  admin_password                  = "hive"
```

Comment:
```suggestion
```
These can be removed since they're not in use.

Reply:
I put additional comments about it, according to @pdmthorsrud request https://github.com/fredrikhgrelland/terraform-nomad-hive/pull/53#discussion_r520382737